compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUOps.td

```diff
@@ -275,6 +275,14 @@ def IREEGPU_CoalescedGatherDMAOp : Op<IREEGPU_Dialect, "coalesced_gather_dma", [
     * When the source is a memref, the operation writes the gathered data into
       a destination out memref operand.
 
+    Each index in the indices operand specifies a location in the source to load,
+    and the load width can be 32, 64, or 128 bits, limited by the underlying hardware.
+    The load width is inferred from the size relationship between the indices
+    and result types. So the result tensor/memref should satisfy the following
+    restriction:
+    * `sizeof(result) == num_indices * load_width`, where load_width is in {4, 8, 16} bytes.
```

Comment:
Where is load width defined? It's not a part of the operation, is it inferred from the types?

Reply:
If so, we should document that